[2023-11-25T20:27:29.000+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: outcomes_dag.transform scheduled__2023-11-19T00:00:00+00:00 [queued]>
[2023-11-25T20:27:29.176+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: outcomes_dag.transform scheduled__2023-11-19T00:00:00+00:00 [queued]>
[2023-11-25T20:27:29.176+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-11-25T20:27:29.193+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): transform> on 2023-11-19 00:00:00+00:00
[2023-11-25T20:27:29.198+0000] {standard_task_runner.py:57} INFO - Started process 96 to run task
[2023-11-25T20:27:29.202+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'outcomes_dag', 'transform', 'scheduled__2023-11-19T00:00:00+00:00', '--job-id', '8', '--raw', '--subdir', 'DAGS_FOLDER/outcomes_dag.py', '--cfg-path', '/tmp/tmp3v72nheu']
[2023-11-25T20:27:29.206+0000] {standard_task_runner.py:85} INFO - Job 8: Subtask transform
[2023-11-25T20:27:29.261+0000] {task_command.py:416} INFO - Running <TaskInstance: outcomes_dag.transform scheduled__2023-11-19T00:00:00+00:00 [running]> on host 17ac1e2d02c6
[2023-11-25T20:27:29.350+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='outcomes_dag' AIRFLOW_CTX_TASK_ID='transform' AIRFLOW_CTX_EXECUTION_DATE='2023-11-19T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-11-19T00:00:00+00:00'
[2023-11-25T20:27:29.784+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/pandas/core/indexes/base.py", line 3653, in get_loc
    return self._engine.get_loc(casted_key)
  File "pandas/_libs/index.pyx", line 147, in pandas._libs.index.IndexEngine.get_loc
  File "pandas/_libs/index.pyx", line 176, in pandas._libs.index.IndexEngine.get_loc
  File "pandas/_libs/hashtable_class_helper.pxi", line 7080, in pandas._libs.hashtable.PyObjectHashTable.get_item
  File "pandas/_libs/hashtable_class_helper.pxi", line 7088, in pandas._libs.hashtable.PyObjectHashTable.get_item
KeyError: 'Name'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 192, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 209, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/etl_scripts/transform.py", line 36, in transform_data
    data = prep_data(data)
  File "/opt/airflow/dags/etl_scripts/transform.py", line 51, in prep_data
    data['name'] = data['Name'].str.replace("*","",regex=False)
  File "/home/airflow/.local/lib/python3.8/site-packages/pandas/core/frame.py", line 3761, in __getitem__
    indexer = self.columns.get_loc(key)
  File "/home/airflow/.local/lib/python3.8/site-packages/pandas/core/indexes/base.py", line 3655, in get_loc
    raise KeyError(key) from err
KeyError: 'Name'
[2023-11-25T20:27:29.798+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=outcomes_dag, task_id=transform, execution_date=20231119T000000, start_date=20231125T202729, end_date=20231125T202729
[2023-11-25T20:27:29.812+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 8 for task transform ('Name'; 96)
[2023-11-25T20:27:29.859+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-11-25T20:27:29.881+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-11-26T00:21:44.129+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: outcomes_dag.transform scheduled__2023-11-19T00:00:00+00:00 [queued]>
[2023-11-26T00:21:44.141+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: outcomes_dag.transform scheduled__2023-11-19T00:00:00+00:00 [queued]>
[2023-11-26T00:21:44.142+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-11-26T00:21:44.158+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): transform> on 2023-11-19 00:00:00+00:00
[2023-11-26T00:21:44.164+0000] {standard_task_runner.py:57} INFO - Started process 104 to run task
[2023-11-26T00:21:44.167+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'outcomes_dag', 'transform', 'scheduled__2023-11-19T00:00:00+00:00', '--job-id', '12', '--raw', '--subdir', 'DAGS_FOLDER/outcomes_dag.py', '--cfg-path', '/tmp/tmpcv1yfjjy']
[2023-11-26T00:21:44.171+0000] {standard_task_runner.py:85} INFO - Job 12: Subtask transform
[2023-11-26T00:21:44.223+0000] {task_command.py:416} INFO - Running <TaskInstance: outcomes_dag.transform scheduled__2023-11-19T00:00:00+00:00 [running]> on host decfc1fe5c09
[2023-11-26T00:21:44.303+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='outcomes_dag' AIRFLOW_CTX_TASK_ID='transform' AIRFLOW_CTX_EXECUTION_DATE='2023-11-19T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-11-19T00:00:00+00:00'
[2023-11-26T00:21:45.108+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-11-26T00:21:45.126+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=outcomes_dag, task_id=transform, execution_date=20231119T000000, start_date=20231126T002144, end_date=20231126T002145
[2023-11-26T00:21:45.165+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-11-26T00:21:45.208+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-11-26T01:52:45.883+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: outcomes_dag.transform scheduled__2023-11-19T00:00:00+00:00 [queued]>
[2023-11-26T01:52:45.896+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: outcomes_dag.transform scheduled__2023-11-19T00:00:00+00:00 [queued]>
[2023-11-26T01:52:45.897+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-11-26T01:52:45.916+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): transform> on 2023-11-19 00:00:00+00:00
[2023-11-26T01:52:45.923+0000] {standard_task_runner.py:57} INFO - Started process 141 to run task
[2023-11-26T01:52:45.928+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'outcomes_dag', 'transform', 'scheduled__2023-11-19T00:00:00+00:00', '--job-id', '12', '--raw', '--subdir', 'DAGS_FOLDER/outcomes_dag.py', '--cfg-path', '/tmp/tmpxlk9ekj2']
[2023-11-26T01:52:45.934+0000] {standard_task_runner.py:85} INFO - Job 12: Subtask transform
[2023-11-26T01:52:45.996+0000] {task_command.py:416} INFO - Running <TaskInstance: outcomes_dag.transform scheduled__2023-11-19T00:00:00+00:00 [running]> on host 2011df1a4519
[2023-11-26T01:52:46.097+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='outcomes_dag' AIRFLOW_CTX_TASK_ID='transform' AIRFLOW_CTX_EXECUTION_DATE='2023-11-19T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-11-19T00:00:00+00:00'
[2023-11-26T01:52:46.827+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-11-26T01:52:46.841+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=outcomes_dag, task_id=transform, execution_date=20231119T000000, start_date=20231126T015245, end_date=20231126T015246
[2023-11-26T01:52:46.873+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-11-26T01:52:46.914+0000] {taskinstance.py:2778} INFO - 2 downstream tasks scheduled from follow-on schedule check
[2023-11-26T02:41:46.809+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: outcomes_dag.transform scheduled__2023-11-19T00:00:00+00:00 [queued]>
[2023-11-26T02:41:46.833+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: outcomes_dag.transform scheduled__2023-11-19T00:00:00+00:00 [queued]>
[2023-11-26T02:41:46.843+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-11-26T02:41:46.882+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): transform> on 2023-11-19 00:00:00+00:00
[2023-11-26T02:41:46.925+0000] {standard_task_runner.py:57} INFO - Started process 128 to run task
[2023-11-26T02:41:46.954+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'outcomes_dag', 'transform', 'scheduled__2023-11-19T00:00:00+00:00', '--job-id', '18', '--raw', '--subdir', 'DAGS_FOLDER/outcomes_dag.py', '--cfg-path', '/tmp/tmpi51f2s9q']
[2023-11-26T02:41:46.979+0000] {standard_task_runner.py:85} INFO - Job 18: Subtask transform
[2023-11-26T02:41:47.163+0000] {task_command.py:416} INFO - Running <TaskInstance: outcomes_dag.transform scheduled__2023-11-19T00:00:00+00:00 [running]> on host d8c33f99ee50
[2023-11-26T02:41:47.356+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='outcomes_dag' AIRFLOW_CTX_TASK_ID='transform' AIRFLOW_CTX_EXECUTION_DATE='2023-11-19T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-11-19T00:00:00+00:00'
[2023-11-26T02:42:00.557+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-11-26T02:42:00.581+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=outcomes_dag, task_id=transform, execution_date=20231119T000000, start_date=20231126T024146, end_date=20231126T024200
[2023-11-26T02:42:00.651+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-11-26T02:42:00.694+0000] {taskinstance.py:2778} INFO - 4 downstream tasks scheduled from follow-on schedule check
[2023-12-17T15:11:16.299+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: outcomes_dag.transform scheduled__2023-11-19T00:00:00+00:00 [queued]>
[2023-12-17T15:11:16.309+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: outcomes_dag.transform scheduled__2023-11-19T00:00:00+00:00 [queued]>
[2023-12-17T15:11:16.309+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-12-17T15:11:16.326+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): transform> on 2023-11-19 00:00:00+00:00
[2023-12-17T15:11:16.333+0000] {standard_task_runner.py:57} INFO - Started process 574 to run task
[2023-12-17T15:11:16.338+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'outcomes_dag', 'transform', 'scheduled__2023-11-19T00:00:00+00:00', '--job-id', '115', '--raw', '--subdir', 'DAGS_FOLDER/outcomes_dag.py', '--cfg-path', '/tmp/tmp5sercdou']
[2023-12-17T15:11:16.343+0000] {standard_task_runner.py:85} INFO - Job 115: Subtask transform
[2023-12-17T15:11:16.398+0000] {task_command.py:416} INFO - Running <TaskInstance: outcomes_dag.transform scheduled__2023-11-19T00:00:00+00:00 [running]> on host 7f0547651eb4
[2023-12-17T15:11:16.484+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='outcomes_dag' AIRFLOW_CTX_TASK_ID='transform' AIRFLOW_CTX_EXECUTION_DATE='2023-11-19T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-11-19T00:00:00+00:00'
[2023-12-17T15:11:36.856+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-12-17T15:11:36.869+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=outcomes_dag, task_id=transform, execution_date=20231119T000000, start_date=20231217T151116, end_date=20231217T151136
[2023-12-17T15:11:36.921+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-12-17T15:11:36.944+0000] {taskinstance.py:2778} INFO - 4 downstream tasks scheduled from follow-on schedule check
